packages/discord.js/src/structures/Webhook.js

```diff
@@ -59,7 +58,7 @@ class Webhook {
        * The type of the webhook
        * @type {WebhookType}
        */
-      this.type = WebhookType[data.type];
+      this.type = data.type;
```

Comment:
the typeguards in this class (`isChannelFollower` and `isIncoming`) need to be updated (again)